shell-native.nix

```diff
@@ -7,6 +7,7 @@ stdenv.mkDerivation rec {
     binutils
     haskell.compiler.ghc822
     pkgconfig
+    postgresql
```

Comment:
Sorry for this, I have seen this line in `default.nix` and was sure that was enough.

Reply:
It's ok, also even with this `stack --nix` has some problems with tests (issue [DSCP-462](https://issues.serokell.io/issue/DSCP-462)).

P.s. I saw your comment on YouTrack, but haven't tried yet.